### DISCUSSION TITLE
Error when build from code

### DISCUSSION THREAD

**Discussion by MaGuanqun** [jimy-byerley/pymadcad#78] [Q&A] [answered]
Hi everyone，

I tried to build the package from the github source code. I just followed the tutorial and built it by `python setup.py build_ext --inplace`.
An error occurred saying "Cannot open source file: 'madcad/core.c' ". Am I doing something wrong to build it from source code

**Comment by jimy-byerley** [comment, marked as answer]
Hello
You might be missing the build dependency `cython`
You can install it with
```shell
pip install cython
```

**Reply by MaGuanqun** [reply]
Thank you!!
